apps/frontend/pages/api/experiments/csv.tsx

```diff
@@ -0,0 +1,26 @@
+import { MongoClient } from 'mongodb';
+
+const mongoCSVHandler = async (req, res) => {
```

Comment:
Switch to follow the pattern that the `[id].tsx` file follows for consistency + ability to copy a download link that doesn't need request body

So this file would move to /pages/api/downloads/csv/[id].tsx` for example

That way a "real link" would be `http://glados.csse.rose-hulman.edu/api/downloads/csv/experimentIdGoesHere`, and you could go to a download link without needing to use request body details to specify what experiment to download